pkg/apis/core/types.go

```diff
@@ -4383,8 +4383,22 @@ type ConfigMap struct {
 
 	// Data contains the configuration data.
 	// Each key must consist of alphanumeric characters, '-', '_' or '.'.
+	// The value must be a string (UTF-8). If the value has byte
+	// sequences not in the UTF-8 range, please use the BinaryData field.
```

Comment:
values with non-UTF-8 byte sequences must use the binaryData field

Reply:
Done